src/serviceView.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SERVICEVIEW_H
+#define SERVICEVIEW_H
+
+#include <QObject>
+#include <QPushButton>
+#include <QLabel>
+#include <QString>
```

Comment:
```suggestion
#include <QIcon>
```
Unused imports, but removing them removes the `QIcon` import so have to add it